### PR TITLE
fix group details path

### PR DESCRIPTION
**Pull request by mgold1234**
# Description

Fixes the link to group details 
Fixes:  
https://issues.redhat.com/browse/THEEDGE-3693
https://issues.redhat.com/browse/THEEDGE-3707

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

File: src/Routes/Groups/GroupTable.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { useHistory, useLocation, useNavigate } from 'react-router-dom';
 import { routes as paths } from '../../constants/routeMapper';
 import { Bullseye, Spinner, Tooltip } from '@patternfly/react-core';
 import { createLink, useFeatureFlags } from '../../utils';
+import inventory from '../Devices/Inventory';
```
Comment (ldjebran) [inline src/Routes/Groups/GroupTable.js:8]: remove this as unused

File: src/Routes/Groups/GroupTable.js [hunk 2]
```diff
@@ -138,7 +141,10 @@ const GroupTable = ({
       cells: [
         {
           title: createLink({
-            pathname: `${paths.fleetManagement}/${ID}`,
+            pathname:
+              currentInventoryPath === 'edge'
+                ? `edge${paths.fleetManagement}/${ID}`
+                : `groups/${ID}`,
```
Comment (ldjebran) [inline src/Routes/Groups/GroupTable.js:147]:
```suggestion
                : `insights${paths.inventory}/groups/${ID}`,
```

**Review by ldjebran — APPROVED** [review]

**Review by acosferreira — APPROVED** [review]
lgtm

**Pull request merged**